README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_sort_table_DOM/)
+    - [DEMO LINK](https://FANTAZER-NURE.github.io/js_sort_table_DOM/)
```

Comment:
https://user-images.githubusercontent.com/38065883/175108337-9c4bec81-bd27-4a1a-a17d-bc955b03516e.mov

It changes table only once for some reason